platforms/bingebot/modes/bingebot-ads.mode.ts

```diff
@@ -1,10 +1,19 @@
 import { startAdEngine } from '@platforms/shared';
-import { DiProcess } from '@wikia/ad-engine';
+import { DiProcess, permutive } from '@wikia/ad-engine';
 import { Injectable } from '@wikia/dependency-injection';
 
 @Injectable()
 export class BingeBotAdsMode implements DiProcess {
 	execute(): void {
-		startAdEngine();
+		const inhibitors = this.callExternals();
+		startAdEngine(inhibitors);
+	}
+
+	private callExternals(): Promise<any>[] {
+		const inhibitors: Promise<any>[] = [];
+
+		permutive.call();
+
+		return inhibitors;
 	}
```

Comment:
```suggestion
		this.callExternals();
		startAdEngine();
	}

	private callExternals(): void {
		permutive.call();
	}
```

There is no need to create `inhibitors` array since we do not push anything to it.